### ISSUE TITLE
Javascript error in Firefox

### ISSUE THREAD

**Issue by steveswinsburg** [sakaiproject/sakai#1332]
_From @payten on March 12, 2015 20:54_

Keyboard navigation etc doesn't work in Firefox! (gradebook-grades.js errors)

     [13/03/2015 6:47:19 am] Steve Swinsburg: TypeError: this.$cell.find(...)[0].innerText is undefined
     [13/03/2015 6:47:27 am] Steve Swinsburg:     columnKey = this.$cell.find(".gb-title, span:first")[0].innerText.trim();
     [13/03/2015 6:47:29 am] Steve Swinsburg: line 551

_Copied from original issue: steveswinsburg/gradebookNG#13_

**Comment by steveswinsburg** [comment]
_From @payten on March 17, 2015 21:38_

Fixed in #14.
